src/amo/css/inc/mixins.scss

```diff
@@ -15,75 +15,83 @@
 
 /* Font mixins. Changes here affect *all* apps */
 @mixin font-light() {
-  font-family: 'Fira Sans', sans-serif;
+  font-family: 'Inter', sans-serif;
+  @supports (font-variation-settings: normal) {
+    font-family: 'Inter Var', sans-serif;
+  }
+
   font-style: normal;
   font-weight: 300;
 }
 
 @mixin font-light-italic() {
```

Comment:
Do we use this mixin? If not, now would be a good time to remove the unused mixins.